api.swagger.yaml

```diff
@@ -5611,6 +5611,82 @@ paths:
           description: The user does not have access to the requested content/comment.
         '404':
           description: No content/comment with that UUID was found.
+  '/rates/{content_id}':
+    parameters:
+      - name: content_id
+        description: The Invotra UUID of content/comment.
+        in: path
+        required: true
+        schema:
+          type: string
+          format: uuid
+          example: '01234567-89ab-cdef-1234-56789abcdef0'
+    put:
+      tags:
+        - rates
+      description: 'Rate a piece of content/comment.'
+      responses:
+        '200':
```

Comment:
Do we need response bodies for the PUT/DELETE here now?

Reply:
@ParryLuke I am not sure what is preferred.
I think it is good for swagger documentation side.
Please let me know your opinion.